connectors/src/connectors/slack/lib/slack_client.ts

```diff
@@ -142,6 +142,10 @@ export async function getSlackUserInfo(
     throw res.error;
   }
 
+  if (!res.user?.profile?.real_name) {
```

Comment:
Assuming we know that it's always defined? 

Side note but if Slack's module types are too loose, io-ts could be a great solution here.